frontend/views/map.js

```diff
@@ -4,10 +4,12 @@ var Backbone = require('backbone'),
     service = require('../models/service'),
     servicetype = require('../models/servicetype'),
     i18n = require('i18next-client'),
-    search = require('../search'),
-    spiderfier = require('OverlappingMarkerSpiderfier/oms')
+    search = require('../search')
 ;
 
+// required here because this is the only view that needs it
+require('marker-spider/dist/oms.min');
```

Comment:
It seems like the practice these days is to save the return value of the `require` and use that to refer to things in that module.